backend/apps/issue_quality/services/quality_scorer.py

```
@@ -0,0 +1,404 @@
+            avg_sentence_length = sum(len(s.sentences) for s in sentences) / len(sentences)
+            
+            # Ideal sentence length: 15-20 words
+            clarity = 1 - min(abs(avg_sentence_length - 17) / 17, 0.5)
+            
+            # Penalize for spelling errors
+            spelling_errors = len(blob.words) - len([w for w in blob.words if w.spellcheck()])
+            spelling_ratio = spelling_errors / (len(blob.words) + 1)
+            clarity = clarity * (1 - min(spelling_ratio, 0.3))
+            
+            return max(0, min(1, clarity))
+        except:
```

Ruff (0.15.21)

[error] Do not use bare `except`

(E722)